### PR TITLE
salmon processes renamed due to monitoring issue

### PR DESCRIPTION
**Pull request by TommYDeeee**
had to rename salmon processes to salmon-relay and salmon-receiver due to monitoring issue

### CODE REVIEW

File: ansible/honeypot.yml [hunk 1]
```diff
@@ -513,6 +513,32 @@
...
+        regexp: '^(.*)python(.*)$' 
+        line: 'python-relay'
+        backrefs: yes
+
+    - name: replace line in relay file
+      lineinfile: 
+        path: "{{ local_home }}/hermes/salmon-receiver/bin/salmon-receiver"
+        regexp: '^(.*)python(.*)$' 
+        line: 'python-receiver'
+        backrefs: yes
```
Comment (TommYDeeee) [inline ansible/honeypot.yml:540]: replacement of python executable to python-receiver

File: ansible/honeypot.yml [hunk 2]
```diff
@@ -513,6 +513,32 @@
...
+
+    - name : Rename salmon receiver file
+      command: mv "{{ local_home }}/hermes/salmon-receiver/bin/salmon" "{{ local_home }}/hermes/salmon-receiver/bin/salmon-receiver"
+
+    - name: replace line in relay file
+      lineinfile: 
+        path: "{{ local_home }}/hermes/salmon-relay/bin/salmon-relay"
+        regexp: '^(.*)python(.*)$' 
+        line: 'python-relay'
+        backrefs: yes
```
Comment (TommYDeeee) [inline ansible/honeypot.yml:533]: replacement of python executable to python-relay

**Review by Larrax — APPROVED** [review]

File: ansible/honeypot.yml [hunk 3]
```diff
@@ -513,6 +513,32 @@
         path: "{{ local_home }}/hermes-git"
         state: absent
 
+    - name : Rename python relay file
+      command: mv "{{ local_home }}/hermes/salmon-relay/bin/python" "{{ local_home }}/hermes/salmon-relay/bin/python-relay"
+
+    - name : Rename python receiver file
+      command: mv "{{ local_home }}/hermes/salmon-receiver/bin/python" "{{ local_home }}/hermes/salmon-receiver/bin/python-receiver"
```
Comment (TommYDeeee) [inline ansible/honeypot.yml:520]: rename python executable file to differentiate them in monitoring system

File: ansible/honeypot.yml [hunk 4]
```diff
@@ -513,6 +513,32 @@
...
+
+    - name : Rename salmon relay file
+      command: mv "{{ local_home }}/hermes/salmon-relay/bin/salmon" "{{ local_home }}/hermes/salmon-relay/bin/salmon-relay"
+
+    - name : Rename salmon receiver file
+      command: mv "{{ local_home }}/hermes/salmon-receiver/bin/salmon" "{{ local_home }}/hermes/salmon-receiver/bin/salmon-receiver"
```
Comment (TommYDeeee) [inline ansible/honeypot.yml:526]: rename salmon exe files to differentiate them

**Comment by schlupov** [comment]
@TommYDeeee please follow our commit message convention next time

    part_of_salmon: what commit changes

so in your case

    ansible: rename the salmon process due to monitoring problems

**Pull request merged**